### DISCUSSION TITLE
Port to Vite

### DISCUSSION THREAD

**Discussion by pedro-pmda** [badeball/cypress-cucumber-preprocessor#992] [Ideas]
I would like to be able to use this Gherkin preprocessor in Vite. Are you thinking of port it?
Thank you so much
Greetings

**Comment by badeball** [comment]
Are you referring to writing a plugin, as described [here](https://github.com/mammadataei/cypress-vite/issues/30#issuecomment-1364548803)? If so, then yes, I've thought about it and that's about it.  Anyone else should feel free to pick this up.

**Reply by badeball** [reply]
FYI, *very* few people need this. Can it be that you're simply using Vite as a dev server for local development? If so, then you don't need a Vite plugin for processing feature files.

**Reply by pedro-pmda** [reply]
Thank you for answering me so fast. I want it to be incorporated into a CI pipe to run the e2e via Electron.

**Comment by badeball** [comment]
FYI, Vite is now supported since v7.1.0.

**Reply by pedro-pmda** [reply]
Thank you very much. I really appreciate this.
